---
layout: default
title: research
---

<h1>Postdoctoral Research</h1>

<div class="justify">

</div>

<h1>Graduate Research</h1>
<div class="justify">
<h2>Overview</h2>
<p>
Broadly, my research in the McKay Lab focused on the epigenetic regulation of
tissue- and temporal-specific gene expression profiles during development.
My work used genomics to develop a high-level understanding
of biological processes and combined it with targeted molecular biology experiments to
validate and expand upon specific questions.
</p>
<p> 
More specifically, I researched how ecdysone hormone signaling in the fruit fly, 
<i>Drosophila melanogaster</i>, elicits different metamorphic responses throughout
the animal and over time. Canonically, it had been thought that pulses of ecdysone 
triggered gene expression cascades in tissues throughout the animal, acting to both
synchronize and drive delopmental transitions forward when the animal was ready.
</p>
<p>
Molecularly, ecdysone achieves this effect by binding to its receptor, EcR,
that, in turn, activates the expression of additional, primary response
transcription factors. Canonically, EcR had always been positioned at the top of
this hierarchy. My research showed that, EcR acted much more broadly than had
previously been believed--it directly bound thousands of enhancers and promoters
throughout the genome. By cloning and imaging the expression of patterns of its
binding sites both over time, and during acute depletion of EcR, we we able to
show that the binding was functional and regulated both the spatial and temporal
activity of these enhancers.
</p>
<p>

</div>

<h2>Questions:</h2>
<ul class='double' style='line-height:1.5'> 
	<li>How does the ecdysone receptor (EcR) coordinate temporal- and tissue-
		specific gene expression responses?</li>
		<ul><li>Is its binding dynamic across time and between tissues?</li></ul>
	<li>How does EcR regulate individual enhancers?</li>
		<ul><li>Does it regulate their temporal activity?</li></ul>
	<li>What dictates where EcR binds?</li>
</ul>
<br>

<figure>
	<img src="/assets/images/fig_1_deltaC_pattern.png" alt="Delta Enhancer Pattern" class='figure'>
	<figcaption class='figure'><b>A cool enhancer I cloned</b><br>
		This enhancer from the <i>Delta (Dl)</i> locus is beautifully patterned.
		A) Browser shot of EcR CUT&RUN from <i>Drosophila</i> wings at the <i>Delta</i> locus. <i>Dl<sup>SOP</sup></i> is highlighted.
		B) <i>Dl<sup>SOP</sup></i> driving nls::GFP (green) compared to <i>Ac</i>, a marker of proneural clusters.
		C) <i>Dl<sup>SOP</sup></i> driving destabalized GFP (dsGFP) compared to <i>Dl</i> protein. When driving dsGFP, 
			<i>Dl<sup>SOP</sup></i> is active only in a few cells that we believe correspond to the sensory organ precursor.
			You can see this figure, and other cool ones <a href='https://www.pnas.org/content/116/20/9893' class='extLink'>here</a>.
	</figcaption>
</figure>


<!--
The basis for my project was a landmark
paper by <a href=https://www-sciencedirect-com.libproxy.lib.unc.edu/science/article/pii/S1534580713006060?via%3Dihub" class = extLink>McKay and Lieb in 2013</a>
in which they showed that the leg, wing, and haltere imaginal discs had chromatin accessibility profiles that were nearly identical.
-->

